nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -367,7 +368,7 @@ in {
 
         "/etc/systemd/system.conf".text = ''
           [Manager]
-          DefaultEnvironment=PATH=/bin:/sbin
+          DefaultEnvironment=PATH=/bin:/sbin ${optionalString (isBool cfg.emergencyAccess && cfg.emergencyAccess) "SYSTEMD_SULOGIN_FORCE=1"}
```

Comment:
Does this cause stage 2 init to have this environment variable set as well?

Reply:
Does not seem to be the case, my services don't have the variable set